CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
-
+### Fixed
+- Escape slashes in entity keys when they appear in the URL path
```

Comment:
Note: if you see the changelog entries, we add also author names for important changes. So feel free to add yourself to this line :)